pkg/telemetry/prometheus/node.go

```diff
@@ -77,7 +78,8 @@ func GetUpdatedNodeStats(prev *livekit.NodeStats, prevAverage *livekit.NodeStats
 		packetsInNow != prevAverage.PacketsIn ||
 		packetsOutNow != prevAverage.PacketsOut ||
 		retransmitBytesNow != prevAverage.RetransmitBytesOut ||
-		retransmitPacketsNow != prevAverage.RetransmitPacketsOut {
+		retransmitPacketsNow != prevAverage.RetransmitPacketsOut ||
+		participantJoinNow != prevAverage.ParticipantJoin {
```

Comment:
we should not recompute average based on participantJoin stats. this counter, like nack & pli counters, is real-time, and not part of the stats update cycle. having it trigger average computation could result in avg bitrate == 0 when performed between stats updates.

Reply:
Good :eye:. Will fix that in a PR now.